py/tests/integration/test_spend_tx.py

```diff
@@ -37,7 +38,6 @@ def test_successful():
     bob_balance0 = {"balance":0} 
     try:
         bob_balance0 = alice_api.get_account_balance(pub_key=test_settings["spend_tx"]["bob_pubkey"])
-        assert_equals(bob_balance1.balance, bob_balance0.balance)
 
     except ApiException as e:
         assert_equals(e.status, 404) # Bob has no account yet
```

Comment:
This assertion may not be called in case the line `alice_api.get_account_balance(pub_key=test_settings["spend_tx"]["bob_pubkey"])` does not throw exception.

It looks like `nose.tools.assert_raises` with context manager shall be used. See [ref1](https://stackoverflow.com/questions/10716506/where-is-noses-assert-raises-function) and [ref2](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises).

Not needed for this PR though.

Reply:
At lines 59-63 (check that Bob's balance has not changed, under the assumption that no block mined in the meantime) I see similar pattern, and there it is relevant as it is part of the user story ("The debit/credit should not happen until the transaction is confirmed, e.g. included in at least one block in the chain."). The assumption that no block mined in the meantime seems fragile (I assume mining autostart is true).

If correct, please track this in a ticket so that we do not lose track of it.

Reply:
There is only one node involved there: Alice's: her node mines. Bob has no node at all so if even if a block is mined - Bob should not receive a reward for it (because it would be mined by Alice). 
WRT exception case - I'll ammend the commit